README.md

```diff
@@ -50,6 +50,22 @@ model = torchao.autoquant(torch.compile(model, mode='max-autotune'))
 
 We also provide a developer facing API so you can implement your own quantization algorithms so please use the excellent [HQQ](https://github.com/pytorch/ao/tree/main/torchao/prototype/hqq) algorithm as a motivating example.
 
+### Evaluation
+
+You can also use the EleutherAI [LM evaluation harness](https://github.com/EleutherAI/lm-evaluation-harness) to directly evaluate models
+quantized with post training quantization, by following these steps:
+
+1. Quantize your model with a [post training quantization strategy](#post-training-quantization).
+2. Upload your quantized model to huggingface ([instructions](https://huggingface.co/docs/hub/en/models-uploading)).
```

Comment:
we can link to huggingface torchao page for these I think

Reply:
I didn't see any details on uploading the quantized model in the torchao page of HF docs, am I looking in the wrong place? https://huggingface.co/docs/transformers/main/en/quantization/torchao?torchao=manual#torchao

Reply:
oh, we should add here: https://huggingface.co/docs/transformers/main/en/quantization/torchao?torchao=manual#serialization

Reply:
I see, so the expected workflow is to serialize and save to disk rather than upload to HF. Updated accordingly.